docs/web/Errors.md

```diff
@@ -0,0 +1,97 @@
+# Errors
+
+Similar to the familiar "404 Not Found" and "500 Internal Server Error" status codes you may have seen in HTTP,Dubbo uses a set of 16 error codes.In the Dubbo protocol, an error is always represented as JSON, and is easily readable in the developer tools of your browser. For example:
+
+```
+HTTP/1.1 400 Bad Request
+Content-Type: application/json
+
+{
+  "code": "invalid_argument",
+  "message": "sentence cannot be empty"
+}
+```
```

Comment:
The following is the correct error format:

HTTP/1.1 400 Bad Request
Content-Type: application/json
Content-Length: 46
{"status":20,"message":"request format error"}